adafruit_ads1x15/ads1015.py

```diff
@@ -61,6 +61,5 @@ def _data_rate_default(self) -> int:
         return 1600
 
     def _conversion_value(self, raw_adc: int) -> int:
-        raw_adc = raw_adc.to_bytes(2, "big")
-        value = struct.unpack(">h", raw_adc)[0]
+        value = struct.unpack(">h", raw_adc.to_bytes(2, "big"))[0]
```

Comment:
Was this (and the other similar change) combined for any particular reason?

Reply:
`raw_adc` is type `int`, line 64 converts it to bytes, mypy complained. I didn't see a good reason to break it out so I combined the lines.

The other option would be to create a new variable.